style.css

```diff
@@ -0,0 +1,446 @@
+@import url("https://fonts.googleapis.com/css2?family=Delicious+Handrawn&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@200;600&display=swap");
+
+body {
+  background-image: url(Bilder/purple_jellyfish.jpg);
+  background-repeat: repeat;
+  background-position: top;
+  background-attachment: fixed;
+  text-align: left;
+  margin: 0;
+}
+
+/*Schriften*/
+p {
+  font-size: 16px;
+  color: #ffffff;
+  font-family: "Montserrat", sans-serif;
+  padding: 20px;
+  text-align: left;
+  margin: 0;
+}
+
+h5 {
+  font-size: 16px;
+  color: #ffffff;
+  font-family: "Montserrat", sans-serif;
+  padding: 0px;
+  text-align: left;
+  margin: 0;
+}
+
+h1 {
+  font-size: 68px;
+  color: #ffcc48ff;
+  font-family: "Delicious Handrawn", cursive;
+  text-align: center;
+  padding: 5px 30px;
+  margin: 0px;
+  background-image: linear-gradient(to left, #ffcc48ff, #6422b8);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+h2 {
+  font-size: 24px;
+  color: #ffffff;
+  font-family: "Montserrat", sans-serif;
+  font-style: SemiBold 600;
+  text-align: center;
+  padding: 5px 30px;
+  margin: 0px;
+}
+
+h3 {
+  font-size: 22px;
+  color: #ffcc48ff;
+  font-family: "Montserrat", sans-serif;
+  font-style: SemiBold 600;
+  text-align: center;
+  padding: 20px 20px 0px;
+  margin: 2px;
+}
+
+h4 {
+  font-size: 16px;
+  color: #ffffff;
+  font-family: "Montserrat", sans-serif;
+  font-style: SemiBold 600;
+  text-align: center;
+  margin: 20px 0px 0px 0px;
+  padding: 0px 20px;
+}
+
+/*Tabelle*/
+th,
+td {
+  border: none;
+  font-family: "Montserrat", sans-serif;
+  color: #ffffff;
+  padding: 0px 25px;
+  font-size: 16px;
+}
+
+table {
+  display: table;
+  border-color: #ffcc48;
+  table-layout: fixed;
+  padding: 20px;
+}
+
+/*flex boxen*/
+
+.flex {
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-intro {
+  align-items: center;
+  justify-content: center;
+  margin: 0px;
+  background-color: black;
+  flex-direction: column;
+  display: flex;
+}
```

Comment:
Is flexbox really needed here?

And PS. It's good practice to add the display: flex; property before the other flex properties

Reply:
Also, I see you've used flexbox on a lot of different elements. How about creating another class with all those properties. Since HTML elements can have more than one class name, that might make your CSS file shorter and more following DRY principle ("Don't repeat yourself").